test/e2e/quadlet/basic.pod

```diff
@@ -1,9 +1,9 @@
 ## assert-key-is Unit RequiresMountsFor "%t/containers"
 ## assert-key-is Service Type forking
 ## assert-key-is Service SyslogIdentifier "%N"
-## assert-key-is-regex Service ExecStartPre ".*/podman pod create --infra-conmon-pidfile=%t/%N.pid --pod-id-file=%t/%N.pod-id --exit-policy=stop --replace --infra-name systemd-basic-infra --name systemd-basic"
-## assert-key-is-regex Service ExecStart ".*/podman pod start --pod-id-file=%t/%N.pod-id"
-## assert-key-is-regex Service ExecStop ".*/podman pod stop --pod-id-file=%t/%N.pod-id --ignore --time=10"
-## assert-key-is-regex Service ExecStopPost ".*/podman pod rm --pod-id-file=%t/%N.pod-id --ignore --force"
+## assert-key-is-regex Service ExecStartPre ".*/podman pod create --infra-conmon-pidfile=%t/%N.pid --exit-policy=stop --replace --infra-name systemd-basic-infra --name systemd-basic"
+## assert-key-is-regex Service ExecStart ".*/podman pod start systemd-basic"
```

Comment:
The container uses `systemd-%N` while the pod uses the full name is the correct?

Reply:
I did not change how the names are generated. In the current `main` branch:
The container name is generated here: https://github.com/containers/podman/blob/4e3226cf1a156a0b6c656644e46640f3c6fe3ac0/pkg/systemd/quadlet/quadlet.go#L908
The pod name is generated here: 
https://github.com/containers/podman/blob/4e3226cf1a156a0b6c656644e46640f3c6fe3ac0/pkg/systemd/quadlet/quadlet.go#L1497

Should I align the pod name with this convention?